doc/client_lua_api.md

```diff
@@ -827,9 +843,25 @@ Call these functions only at load time!
 Class reference
 ---------------
 
+### ModChannel
+
+An interface to use mod channels on client and server
+
+#### Methods
+* `leave()`: leave the mod channel.
+    * Client leaves channel `channel_name`. 
+    * No more incoming or outgoing messages can be sent to this channel from client mods.
+    * This invalidate all future object usage
+    * Ensure your set mod_channel to nil after that to free Lua resources
```

Comment:
`Make sure to set mod_channel to nil afterwards in order to free Lua resources`

not sure this is particularly notable for this user data however, it should be done everywhere